[WIP] DVC: Add new page and for several sub commands (#4225)
tldr-pages/tldr: pages/common/dvc-gc.md

# dvc gc

> > Remove unused files and directories from cache or remote storage.

Reviewer: Perhaps this will read better?
```suggestion
> Remove unused files and directories from the cache or remote storage.
```

> More information: <https://dvc.org/doc/command-reference/gc>.

> - Garbage collect from cache, keeping only version referenced by current workspace:

Reviewer: Perhaps this will read better?
```suggestion
- Garbage collect from the cache, keeping only versions referenced by the current workspace:
```

`dvc gc --workspace`

> - Garbage collect from cache, keeping only version referenced by branch/tags/all_commits:

Reviewer: Perhaps this will read better?
```suggestion
- Garbage collect from the cache, keeping only versions referenced by branch, tags, and commits:
```

`dvc gc --all-branches --all-tags --all-commits`

> - Garbage collect from cache, including default cloud remote storage (if set):

Reviewer: Perhaps this will read better?
```suggestion
- Garbage collect from the cache, including the default cloud remote storage (if set):
```

`dvc gc --all-commits --cloud`

> - Garbage collect from cache, including specific cloud remote storage:

Reviewer: Perhaps this will read better?
```suggestion
- Garbage collect from the cache, including a specific cloud remote storage:
```

> `dvc gc --all-commits --cloud -r {{remote_name}}`

Reviewer: Is there a long-form option for `-r` here? We generally prefer long-form arguments in tldr pages.

Author: Yes, good catch. Will update 😄
